monitoring/oplog-populator/alerts.yaml

```diff
@@ -0,0 +1,58 @@
+x-inputs:
+- name: namespace
+  type: constant
+  value: zenko
+- name: oplog_populator_job
+  type: config
+  value: artesca-data-backbeat-oplog-populator-headless
+- name: connectorReconfigurationFailureThreshold
+  type: config
+  value: 0.2
+- name: oplogPopulatorChangeStreamLagThreshold
+  type: config
+  value: 2
+
+groups:
+- name: Oplog Populator
+  rules:
+
+  - alert: OplogPopulatorUnavailable
+    Expr: |
+      sum(up{namespace="${namespace}",job="${oplog_populator_job}"}) < 1
+    For: "30s"
+    Labels:
+     severity: critical
+    Annotations:
+      zenko_service: backbeat-oplog-populator
+      description: "Oplog populator pod is down"
+      summary: "Oplog populator service is in critical state"
+
+  - alert: KafkaConnectFailedConnectorConfiguration
+    Expr: |
+        (sum (rate(oplog_populator_reconfiguration{success="false",job="${oplog_populator_job}",namespace="${namespace}"}[1m]))
+        / sum (rate(oplog_populator_reconfiguration{job="${oplog_populator_job}",namespace="${namespace}"}[1m])))
+        >= ${connectorReconfigurationFailureThreshold}
+    For: "1m"
+    Labels:
+     severity: critical
+    Annotations:
+      zenko_service: backbeat-oplog-populator
+      description: "Oplog populator failed to configure connectors"
+      summary: "Oplog populator couldn't update kafka connect connectors"
+
+  - alert: OplogPopulatorMetastoreChangeStreamLagThreshold
+    Expr: |
+        sum(rate(
```

Comment:
instead of checking the average, we should prbably compare the 95th or 99th quantile